app/src/main/res/values/dimens.xml

```diff
@@ -2,4 +2,7 @@
 <resources>
     <dimen name="button_main_inset_vertical_size_xsmall">10dp</dimen>
     <dimen name="text_size_strings_normal">20sp</dimen>
+    <dimen name="dialog_fragment_margin_size_small">20dp</dimen>
+    <dimen name="dialog_fragment_padding_size_xsmall">10dp</dimen>
+    <dimen name="dialog_fragment_layout_height_xxbig">200dp</dimen>
```

Comment:
these resources are for all dialog fragments?

Reply:
yes

Reply:
ok, which one? because you only got one

Reply:
good point, fixed!